index.html

```diff
@@ -62,5 +62,184 @@ <h1>Hi👋, I'm Cephas Tokoli</h1>
          </div>     
         </div>   
     </section>
+    <section class="grid">
+            <div>
+                <h2 class="headings">Projects</h2>
+            </div>
+     
+            <div class="project">
+                <div class="laptop-img">
+                    <img src="images/laptop.png" alt="laptop">
+                </div>
+                <div class="project-details">
+                    <div class="tech-details">
+                        <p class="project-name">Project name goes here</p>
+                    <div class="tech-stack">
+                        <p>HTML</p>
+                        <p>Ruby on Rails</p>
+                        <p>Javascript</p>
+                    </div>
+                    <div class="details-button">
+                       <div>
+                        <p >See this project</p>
+                        <img src="images/Union.svg" alt="union">
+                       </div>
```

Comment:
- [ ] Kindly make sure that the button (e.g., "See this project") is a link `a` or `button` tag ⚒
- [ ] Also, I think it would be better to reduce the width of the `button` a little bit to match the given design. See the images below 👇

Your design | Figma design
---| ---|
![image](https://user-images.githubusercontent.com/106119105/236016908-7c00b527-76db-4f63-bc16-09a4aaabc1a2.png) | ![image](https://user-images.githubusercontent.com/106119105/236016765-c9ce1509-d328-409d-a958-375fd4deb934.png)